django/contrib/admin/utils.py

```diff
@@ -481,23 +481,3 @@ def remove_trailing_data_field(fields):
     except NotRelationField:
         fields = fields[:-1]
     return fields
-
-
-def get_limit_choices_to_from_path(model, path):
```

Comment:
I come to the conclusion that get_limit_choices_to_from_path() always returns empty models.Q(). The reason is that the get_limit_choices_to() is looked from field.rel, but Rel objects do not have get_limit_choices_to() method, only Field instances have that.